buildspec_canary.yml

```diff
@@ -6,8 +6,9 @@ phases:
       golang: 1.13
   build:
     commands:
-      - latestTag=$(git describe --tags `git rev-list --tags --max-count=1`)
-      - echo "checking out $latestTag"
-      - git checkout $latestTag
+      # - latestTag=$(git describe --tags `git rev-list --tags --max-count=1`)
+      # - echo "checking out $latestTag"
+      # - git checkout $latestTag
```

Comment:
or we keeping them for some reasons 👁️ ?
```suggestion
```

Reply:
I thought it might be easier to revert back once the changes are released 🤷 I'm ok to delete if we prefer to

Reply:
nah it's fine 👍🏼